content/09.overfitting.md

```diff
@@ -8,6 +8,12 @@ To continue the student analogy, a smarter student has greater potential for mem
 
 ![A visual example of overfitting and failure to generalize. While a high-degree polynomial gets high accuracy on its training data, it performs poorly on data unlike that which it has seen before. In contrast, a simple linear regression works well on both datasets. The greater representational capacity of the polynomial is analogous to using a larger or deeper neural network.](images/overfitting.png){#fig:overfitting-fig}
 
+To evaluate deep supervised learning models, they should be trained, tuned, and tested on non-overlapping datasets.
```

Comment:
"deep learning" -> "DL"

Reply:
Not sure it's worth mentioning that if a single dataset is split into 2 or 3 for training, tuning, testing and preprocessing (like quantile normalization) is applied before this split, then the resulting datasets may not be truly independent.

Reply:
@siminab this is a good point but it should go in the overfitting tip.

Reply:
Ok I made a new issue for that

Reply:
Fixed